gpkit/nomials/math.py

```diff
@@ -523,26 +468,27 @@ def as_posyslt1(self, substitutions=None):
             #  monomial indexes pre-substitution, and whose values are the
             #  percentage of the simplified  monomial's coefficient that came
             #  from that particular parent.
-
-            self.pmap = pmap  # pylint: disable=attribute-defined-outside-init
-            try:
-                p = Posynomial(exps, cs, simplify=False)
-            except ValueError:
+            #
+            hmap = self._simplify_posy_ineq(hmap, self.pmap)
+            if hmap is None:
+                continue
+            if all(c == 0 for c in hmap.values()):
+                continue  # skip tautological 0'd constraint
```

Comment:
right, the latter -- I'm wondering why an `hmap` full of zeros would make it this far when we have the spiffy remove_zeros method.